djpsa/halo/records/ticket/model.py

```diff
@@ -1,7 +1,42 @@
+from enum import Enum
+
 from django.db import models
 from model_utils import FieldTracker
 
 
+class ItilRequestType(Enum):
+    # System defined ITIL request types. We can use these to identify
+    # the type of ticket we are dealing with. This is useful for
+    # when an instance has custom ticket types beyond the default.
+    INCIDENT = 1
+    CHANGE_REQUEST = 2
+    SERVICE_REQUEST = 3
+    PROBLEM = 4
+    REQUEST_FOR_QUOTE = 20
+    ADVICE_OTHER = 21
+    PROJECTS = 22
+    TASKS = 23
+
+
+class TicketOnlyManager(models.Manager):
+    def get_queryset(self):
+        return self.get_queryset().filter(itil_requesttype__in=[
```

Comment:
Can this just exclude what ProjectOnlyManager filters for? Otherwise someday we're going to add an ItilRequestType, and forget to add it in this list, and take 2 hours debugging why it's not showing up in results.